.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Run npm build and npm test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [8.x, 9.x, 12.x]
```

Comment:
just curious - why have multiple versions of node here but not in other places?

Reply:
This workflow `build.yml` is just testing that our library works on different versions of node. Our lowest version we support is 8.x so we build and publish our packages with version 8.x